Makefile

```
@@ -25,4 +25,8 @@ test:
 	go test ./...
 
 verify:
-	@hack/verify.sh
\ No newline at end of file
+	@hack/verify.sh
+
+
+test-e2e: build
+	@hack/run-e2e.sh
```

```suggestion

test-e2e: build
	@hack/run-e2e.sh
```

You can remove this extra line and add a new line at the bottom to remove the `No newline at the end of the file` error.